test/src/device-request-prompt.spec.ts

```diff
@@ -60,4 +60,50 @@ describe('device request prompt', function () {
       })
     ).rejects.toThrow(TimeoutError);
   });
+
+  it('can be opened', async function () {
+    this.timeout(1_000);
```

Comment:
Nit: It seems we want to do this for all test, so let's extract it to the first line of `describe('device request prompt'`